dist/spec.yaml

```diff
@@ -9203,7 +9203,7 @@ definitions:
       x5t:
         readOnly: true
         type: string
-      'x5t#S256':
+      x5t#S256:
```

Comment:
Got added automatically.

Reply:
I need to figure out why this goes back and forth. some of us are adding it when building, some are not.  Its automated, so not something we are doing directly